src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,24 @@
 package racingcar;
+import javax.lang.model.element.Name;
+import java.util.Scanner;
+import java.util.ArrayList;
+import static camp.nextstep.edu.missionutils.Console.readLine;
 
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+        ArrayList<Car> cars = new ArrayList<>();
```

Comment:
```suggestion
        List<Car> cars = new ArrayList<>();
```
보통 이렇게 선언해서 사용합니다! 
저번주 강의 보시고 다형성에 대해 공부해보면 좋을 것 같아요 :)